concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -0,0 +1,1047 @@
+                  {{cf_grafana_version}} grafana-boshrelease
+
+      - task: cf-deploy
+        config:
+          platform: linux
+          image: docker:///governmentpaas/bosh-cli
+          inputs:
+            - name: paas-cf
+            - name: cf-manifest
+            - name: bosh-secrets
+          outputs:
+            - name: updated-cf-manifest
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/bosh_login.sh bosh-secrets/bosh-secrets.yml
+                sed -e "s/^director_uuid:.*/director_uuid: $(bosh status --uuid)/" < cf-manifest/cf-manifest.yml > updated-cf-manifest/cf-manifest.yml
+                bosh deployment updated-cf-manifest/cf-manifest.yml
+                bosh -n deploy
+      - put: cf-manifest
+        params:
+          file: updated-cf-manifest/cf-manifest.yml
+
+  - name: post-deploy
+    plan:
+    - aggregate:
+        - get: pipeline-trigger
+          passed: [ 'cf-deploy']
+          trigger: true
+        - get: paas-cf
+          passed: ['cf-deploy']
+        - get: cf-secrets
+          passed: ['generate-cf-config']
+        - get: cf-manifest
+          passed: ['cf-deploy']
+        - get: graphite-nozzle
+    - task: retrieve-config
+      config:
+        platform: linux
+        inputs:
+          - name: paas-cf
+          - name: cf-secrets
+          - name: cf-manifest
+        outputs:
+          - name: config
+        params:
+        image: docker:///ruby#2.2-slim
+        run:
+          path: sh
+          args:
+            - -e
+            - -c
+            - |
+              VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
+
+              CF_ADMIN=admin
+              CF_PASS=$($VAL_FROM_YAML secrets.uaa_admin_password cf-secrets/cf-secrets.yml)
+              FIREHOSE_USER=graphite-nozzle
+              FIREHOSE_PASS=$($VAL_FROM_YAML secrets.uaa_clients_firehose_password cf-secrets/cf-secrets.yml)
+              API_ENDPOINT=$($VAL_FROM_YAML properties.cc.srv_api_uri cf-manifest/cf-manifest.yml)
+              UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+              DOPPLER_ENDPOINT=$($VAL_FROM_YAML properties.loggregator.traffic_controller_url cf-manifest/cf-manifest.yml)
+              GRAPHITE_SERVER=$($VAL_FROM_YAML jobs.graphite_z1.networks.cf1.static_ips.0 cf-manifest/cf-manifest.yml)
+
+              for var_name in CF_ADMIN CF_PASS FIREHOSE_USER FIREHOSE_PASS API_ENDPOINT UAA_ENDPOINT DOPPLER_ENDPOINT GRAPHITE_SERVER; do
+                echo export "${var_name}"=\"$(eval echo \$${var_name})\"
+              done > config/config.sh
+
+    - task: deploy-graphite-nozzle
+      config:
+        platform: linux
+        inputs:
+          - name: graphite-nozzle
+          - name: config
+        params:
+        image: docker:///governmentpaas/cf-acceptance-tests
```

Comment:
Should we create a separate cf-cli container for this?  We don't need the Go runtime, or godep etc for this task.